README.md

```diff
@@ -1,119 +1,85 @@
-ipam-driver
-=============
+# Infoblox Docker IPAM Plugin
 
-Docker (libnetwork) driver for IPAM
------------------------------------
+Infoblox ipam-plugin is a Docker Engine managed plugin that interfaces with Infoblox
+to provide IP Address Management services for the Docker containers.
 
-ipam-driver is a Docker libnetwork driver that interfaces with Infoblox to provide IP Address Management
-services. libnetwork is the library provided by Docker that allows third-party drivers for container
-networking.
+## Prerequisite
 
-```
-NOTE: There is a limitation in Swarm Mode introduced in Docker 1.12 where IPAM plugins are not supported.
-As a result, the ipam-driver cannot be used in that configuration, althought the driver still works in
-a traditional Docker Swarm. It is expected that this limitation will be fixed in an upcoming Docker
-release, and would therefore enable the ipam-driver to work in Swarm Mode.
-```
-
-Prerequisite
-------------
 To use the driver, you need access to the Infoblox DDI product. For evaluation purposes, you can download a
 virtual version of the product from the Infoblox Download Center (https://www.infoblox.com/infoblox-download-center)
 Alternatively, if you are an existing Infoblox customer, you can download it from the support site.
 
-Refer to CONFIG.md on how to configure vNIOS.
+Refer to [CONFIG.md](docs/CONFIG.md) on how to configure vNIOS.
 
-Build
------
-For dependencies and build instructions, refer to ```BUILD.md```.
+## Installation
 
-Installation
-------------
-By default, the ipam-driver assumes that the "Cloud Network Automation" licensed feature is activated. Should
-this not be the case, refer to "Manual Configuration of Cloud Extensible Attributes" in CONFIG.md for additional
+By default, the ipam-plugin assumes that the "Cloud Network Automation" licensed feature is activated in the NIOS. Should this not be the case, refer to "Manual Configuration of Cloud Extensible Attributes" in CONFIG.md for additional
 configuration required.
 
-Run Executable
---------------
-ipam-driver accepts a number of arguments which can be listed by specifying -h:
```

Comment:
Okay if there is a use case for this then I will add this feature. Thanks for the info.